templates/controller-deployment.yaml

```diff
@@ -79,6 +80,10 @@ spec:
           {{- else }}
           value: http://$(HOST_IP):8500
           {{- end }}
+        - name: NAMESPACE
```

Comment:
What is this used for?

Reply:
from a previous iteration of the webhook!